js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
```

Comment:
This file only contains commented code, should try not to commit empty files or commented code.

Reply:
as in the previous comment, please make this comment a bit more polite